src/views/components/attendees-list/attendees/title.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Generic: Success Order Page Attendee list (Your Tickets) - title
```

Comment:
I guess we can remove the `Your Tickets` reference, now.
```suggestion
 * Generic: Success Order Page Attendee list - title
```